src/core/sys/linux/fcntl.d

```diff
@@ -17,6 +17,24 @@ enum {
     FALLOC_FL_UNSHARE_RANGE = 0x40
 }
 
+// From asm-generic/fcntl.h
+/*
```

Comment:
I see that the other definition here isn't marked with a public Ddoc comment, but I don't see any reason why it shouldn't be exposed in the docs?